event_record.rb

```diff
@@ -0,0 +1,161 @@
+require 'date'
+require_relative 'event_details'
+class EventRecord
+  @@events_date = {}
+  @@events_details = {}
+  @@id = 0
+  def add_event_details(name, date)
+    begin
+      date = date.strip.split("-")
+      date = DateTime.new(date[0].to_i, date[1].to_i, date[2].to_i)
+      @@id += 1
+      @@events_details[@@id] = Event_Details.new(@@id, name, date)
+      key = date.year.to_s + "-" + date.month.to_s
+      day = date.day
+      if @@events_date.has_key? (key)
+        @@events_date[key].push([name, day])
+      else
+        @@events_date[key] = Array.new
+        @@events_date[key].push([name, day])
+      end
+      print "Event added\n"
```

Comment:
why we are using print with "\n" and sometimes puts ?
Code should be consistent and try to avoid adding special characters if a built-in method is there for it.

Reply:
done